packages/office-ui-fabric-react/src/utilities/color/cssColor.ts

```diff
@@ -1,42 +1,82 @@
 import { IRGB } from './interfaces';
 import { MAX_COLOR_ALPHA } from './consts';
-import { COLOR_VALUES } from './colorValues';
 import { hsl2rgb } from './hsl2rgb';
 
 /**
  * Converts a valid CSS color string to an RGB color.
  * Note that hex colors *must* be prefixed with # to be considered valid.
  * Alpha in returned color defaults to 100.
+ * Four and eight digit hex values (with alpha) are supported if the current browser supports them.
  */
-export function cssColor(color: string): IRGB | undefined {
+export function cssColor(color?: string): IRGB | undefined {
   if (!color) {
     return undefined;
   }
-  return _named(color) || _hex3(color) || _hex6(color) || _rgba(color) || _hsla(color);
+
+  // Need to check the following valid color formats: RGB(A), HSL(A), hex, named color
+
+  // First check for well formatted RGB(A), HSL(A), and hex formats at the start.
+  // This is for perf (no creating an element) and catches the intentional "transparent" color
+  //   case early on.
+  const easyColor: IRGB | undefined = _rgba(color) || _hex6(color) || _hex3(color) || _hsla(color);
+  if (easyColor) {
+    return easyColor;
+  }
+
+  // if the above fails, do the more expensive catch-all
+  return _browserCompute(color);
 }
 
 /**
- * If `str` is a valid HTML color name, returns an RGB color (with alpha 100).
- * Otherwise returns undefined.
+ * Uses the browser's getComputedStyle() to determine what the passed-in color is.
+ * This assumes _rgba, _hex6, _hex3, and _hsla have already been tried and all failed.
+ * This works by attaching an element to the DOM, which may fail in server-side rendering
+ *   or with headless browsers.
  */
-function _named(str: string): IRGB | undefined {
-  const c = (COLOR_VALUES as any)[str.toLowerCase()];
+function _browserCompute(str: string): IRGB | undefined {
+  if (!document) {
```

Comment:
You'll probably need to do `if (typeof document === 'undefined')` to avoid throwing.